/// # Help with Try Catch Finally
///
/// ## Finally
/// The finally clause runs whether or not an exception is thrown.
///
/// If there is a return statement in the try/catch block, the finally clause is run after the try/catch block operations and before the try/catch block return statement.
/// If there is no catch clause which matches the Exception, the finally clause is run before propagating the Exception.
class HelpTryCatchFinally {}
